gensim/test/test_word2vec.py

```diff
@@ -683,7 +683,10 @@ def test_reset_from(self):
         """Test if exception is raised when reset_from is used"""
         model = word2vec.Word2Vec(sentences, min_count=1)
         model2 = word2vec.Word2Vec(new_sentences, min_count=1)
-        self.assertRaises(AttributeError, model.reset_from(model2))
+        try:
+            model.reset_from(model2)
+        except AttributeError:
+            self.fail('AttributeError in reset_from()')
```

Comment:
Because an uncaught error will result in a test failure, this catch-then-fail seems superfluous - it's enough to try the `reset_from()`, and do some afterwards checking that it's had the intended effect.